code/modules/shuttle/computer.dm

```diff
@@ -305,10 +305,17 @@
 					to_chat(user, SPAN_NOTICE("[src]'s screen says \"Unable to launch, fuel insufficient\"."))
 					return
 
-				if(tgui_alert(user, "Early launch the lifeboat?", "Confirm", list("Yes", "No"), 10 SECONDS) == "Yes")
-					to_chat(user, SPAN_NOTICE("[src]'s screen blinks and says \"Early launch accepted\"."))
-					lifeboat.evac_launch()
-					return
+				var/response = tgui_alert(user, "Launch the lifeboat?", "Confirm", list("Yes", "No", "Emergency Launch"), 10 SECONDS)
```

Comment:
need something to prevent spam launching. i don't know if this has undesirable effects on shuttle side (though nothing really prevents it) but at least the announcements would get duplicated

Reply:
thanks, fixed